### PR TITLE
Footer navbar

### PR DESCRIPTION
**Pull request by drowsy810301**
- fix navbar&footer css problem
- index upcoming contest can register

### CODE REVIEW

File: index/templates/index/index.html [hunk 1]
```diff
@@ -118,7 +121,16 @@ <h3 style="display: inline;"><b>Announcements</b></h3>
               <a href="{% url 'contest:contest' c_upcoming.id %}">{{ c_upcoming.cname }}</a>
```
Comment (nickelcheng) [inline index/templates/index/index.html]: Please remove the link when request user can not view the contest. (only admin and contest owner / coowner can view it)

**Comment by nickelcheng** [comment]
+1 besides the comment.

**Comment by drowsy810301** [comment]
comment fixed

File: index/templates/index/index.html [hunk 2]
```diff
@@ -115,10 +118,23 @@ <h3 style="display: inline;"><b>Announcements</b></h3>
         {% for c_upcoming in c_upcomings %}
           <tr>
             <td>
+              {% if user|has_auth:c_upcoming.id and user.has_admin_auth %}
```
Comment (nickelcheng) [inline index/templates/index/index.html]: It should be "or" instead of "and"
```{% if user|has_auth:c_upcoming.id or user.has_admin_auth %}```

**Comment by drowsy810301** [comment]
comment fixed

**Comment by nickelcheng** [comment]
going to merge.

**Pull request merged**